lib/axlsx/workbook/worksheet/comment.rb

```diff
@@ -61,7 +61,6 @@ def ref=(v)
     # @param [String] str
     # @return [String]
     def to_xml_string(str = "")
-      author = @comments.authors[author_index]
```

Comment:
I'm pretty sure this line isn't needed as `author` is already an attribute of this `Comment` class.